modules/about-oadp-update-channels.adoc

```diff
@@ -9,23 +9,25 @@
 
 When you install an OADP Operator, you choose an _update channel_. This channel determines which upgrades to the OADP Operator and to Velero you receive. You can switch channels at any time.
 
-There are three update channels:
+The available update channels are:
 
-* The *stable* channel contains the latest minor updates (y-stream updates) and patches (z-stream updates) of OADP ClusterServiceVersion`. As each new release is published, the available `ClusterServiceVersion` of the OADP Operator will be appended with the latest available minor patch.
+* The *stable* channel is now *deprecated*. The *stable* channel contains the patches (z-stream updates) of OADP `ClusterServiceVersion` for `oadp.v1.1.z` and older versions from `oadp.v1.0.z`.
```

Comment:
If you are just saying that the `stable` channel is deprecated or will no longer be used, I don't think you need bold formatting for "deprecated."

Reply:
@kcarmichael08 thank you for your helpful review.